packages/wmr/src/start.js

```diff
@@ -166,7 +166,6 @@ const injectWmrMiddleware = ({ cwd }) => {
 		} catch (e) {
 			next();
 		}
-		next();
```

Comment:
Wait on second thought: Do we still need to call `next()` in the success case when we don't end the request?

Reply:
Oh right, we do, kinda missed that if 🙈

Reply:
I think this needs to be:

```js
try {
  if (...) {
    res.end(result);
    return;
  }
} catch (e) {}
next();
```

Reply:
Oh, I moved out the if now